src/ToolingSharedLibrary/Includes/CodeGeneration/Flatbuffers/Contants.h

```diff
@@ -12,14 +12,17 @@ static inline constexpr std::string_view c_flatbuffer_compiler_name = "flatc.exe
 
 static inline constexpr std::string_view c_flatbuffer_compiler_default_path = "{}\\flatc.exe";
 
-static inline constexpr std::string_view c_cpp_gen_args = "--cpp --no-prefix --cpp-std c++17 --gen-object-api --force-empty";
+// The flatbuffer compiler generates style warnings if the table fields are not snake case. We'll use
+// --no-warnings to prevent this, since we can't dictate to users that they must use snake
+// case for their function variable names.
```

Comment:
seesm sketchy :P

can you have name conflicts if you have

snake_case_var and SnakeCaseVar?